arches/app/media/js/viewmodels/workflow-step.js

```diff
@@ -182,7 +196,7 @@ define([
         this.getFromLocalStorage = function(key) {
             var allStepsLocalStorageData = JSON.parse(localStorage.getItem(STEPS_LABEL)) || {};
 
-            if (allStepsLocalStorageData[self.id()]) {
+            if (allStepsLocalStorageData[self.id()] && allStepsLocalStorageData[self.id()][key]) {
```

Comment:
This needs to be refactored to return falsey values. eg if someone stored int 0, it wouldn't be returned here.